### PR TITLE
feat(redis-common): span add extension tags

### PR DESCRIPTION
**Pull request by hiColors**
span add extension tags

### CODE REVIEW

**Comment by malafeev** [comment]
@hiColors what is the purpose if this PR? What is the issue?

File: opentracing-redis-common/src/main/java/io/opentracing/contrib/redis/common/TracingHelper.java [hunk 1]
```diff
@@ -19,13 +19,9 @@
...
 import io.opentracing.tag.Tags;
+
 import java.nio.charset.StandardCharsets;
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.Collection;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
+import java.util.*;
```
Comment (malafeev) [inline opentracing-redis-common/src/main/java/io/opentracing/contrib/redis/common/TracingHelper.java]: please don't use "*"

**Comment by hiColors** [comment]
> @hiColors what is the purpose if this PR? What is the issue?

for this https://github.com/opentracing-contrib/java-redis-client/issues/44

in springboot application, sometimes it's has multi redis, we will add extension tags remark different datasource.

File: opentracing-redis-common/src/main/java/io/opentracing/contrib/redis/common/TracingHelper.java (hunk 1 above)
Comment (hiColors) [reply opentracing-redis-common/src/main/java/io/opentracing/contrib/redis/common/TracingHelper.java]: ok, ide smart import, i will fix it.

**Pull request merged**